scripts/plot_gpx.py

```diff
@@ -0,0 +1,109 @@
+shutil.copytree(unpack_path, temp_unpack_path)
+
+slide_path = temp_unpack_path+"/ppt/slides/slide1.xml"
+
+def createPptxFromTrackData(trackData):
+    soup = BeautifulSoup(open(slide_path, 'r').read(), 'xml')
+    shape_tag = soup.find('sp')
+    shape_tag.extract()
+    trackCount = 0
+    for track in trackData:
+        temp_shape_tag = copy.deepcopy(shape_tag)
+
+        path_tag = temp_shape_tag.find('path')
+        # if(trackCount!=0):
+        #     temp_shape_tag.find('off')['x'] = str(int(temp_shape_tag.find('off')['x']) + 2000000)
+            # temp_shape_tag.find('off')['y'] = str(int(temp_shape_tag.find('off')['y']) + 2000000)
+        # 
+        # w = path_tag['w']
+        # h = path_tag['h']
+
+        #Adding coordinates
+        coordinates_detail = track['coordinates']
+        coordinates = []
+        for coordinate_detail in coordinates_detail:
+            coordinates.append(coordinate_detail['coor_set'])
+
+        num_coordinate = 1
+        for coordinate in coordinates:
+            (x,y) = coordinate
+            x = round(float(x))
+            x = x*10000
+            x = int(x)
+            x = str(x)
+            y = round(float(y))
+            y = y*10000
+            y = int(y)
+            y = str(y)
+            if(num_coordinate==1):
+                coordinate_soup = BeautifulSoup("<a:moveTo><a:pt x='"+x+"' y='"+y+"'/></a:moveTo>", 'xml')
+                path_tag.append(coordinate_soup.find('moveTo'))
+                num_coordinate+=1
+            else:
+                coordinate_soup = BeautifulSoup("<a:lnTo><a:pt x='"+x+"' y='"+y+"'/></a:lnTo>", 'xml')
+                path_tag.append(coordinate_soup.find('lnTo'))
+                num_coordinate+=1
+
+        #Adding color to the track
+        colors = track['color']
+        temp = colors.split("[")[1]
+        temp = temp[0:-1]
+        temp = temp.split(",")
+        r = int(temp[0].split("=")[1])
+        b = int(temp[1].split("=")[1])
+        g = int(temp[2].split("=")[1])
+
+        hex = "{:02x}{:02x}{:02x}".format(r,g,b)
+
+        temp_shape_tag.find('srgbClr')['val'] = hex.upper()
+
+        soup.find('spTree').append(temp_shape_tag)
+        trackCount+=1
+
+    soup_text = soup.prettify()
+    soup_text = soup_text.replace("<body>","")
+    soup_text = soup_text.replace("<html>","")
+    soup_text = soup_text.replace("</body>","")
+    soup_text = soup_text.replace("</html>","")
+    soup_text = soup_text.replace("<lnTo>","<a:lnTo>")
+    soup_text = soup_text.replace("</lnTo>","</a:lnTo>")
+    soup_text = soup_text.replace("<moveTo>","<a:moveTo>")
+    soup_text = soup_text.replace("</moveTo>","</a:moveTo>")
+    soup_text = soup_text.replace("<pt","<a:pt")
+    soup_text = soup_text.replace("</pt","</a:pt")
+    soup_text = soup_text.strip()
+    # print soup_text
+    text_file = open(slide_path, "w")
+    text_file.write(soup_text)
+    text_file.close()
+
+    os.system("python scripts/pack.py --unpack_path "+temp_unpack_path)
```

Comment:
@purvil12c = shouldn't this be a call to the Python function?

I appreciate we may have to refactor the pack.py function (again), but it will make our implementation tidier).

Reply:
Sure we can refactor the code.

Reply:
Can we refactor the pack.py function in a different issue altogether later on? I think it will cause conflicts between this branch (issue#8) and issue#5 branch I am working on.

Reply:
I know what you mean, but it's not scary.  

You would switch to this branch, fix the change, and I can accept the PR.

Then you switch to your #5 branch, and merge in Master branch.   Your #5 branch will now include the fixes.

I know that's more work, but I'm keen to maintain the quality of the Master branch.